### ISSUE TITLE
[NBug] An existing connection was forcibly closed by the remote host

### ISSUE THREAD

**Issue by LazarevskaAngela** [gitextensions/gitextensions#7613]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.Net.Sockets.SocketException (0x80004005): An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at System.Net.Sockets.NetworkStream.EndRead(IAsyncResult asyncResult)
```

## Additional information
while downloading update

## Environment
- Git Extensions 3.3.0.7719
- Build e52f5178e8d3c9899efbe6fae66026865afa3fe1
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4075.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Fixed in #7576

**Comment by RussKie** [comment]
Please update to 3.3.1
